hybrid/power_source.py

```diff
@@ -36,7 +36,15 @@ def __init__(self, name, site: SiteInfo, system_model, financial_model):
         self._financial_model = financial_model
         self._layout = None
         self._dispatch = PowerSourceDispatch
+
```

Comment:
Can you update the docstring of __init__ to explain this?

Reply:
Done